Bender.yml

```diff
@@ -0,0 +1,110 @@
+package: 
+  name: pulp_soc
+  authors:
+    - "Robert Balas <balasr@iis.ee.ethz.ch>"
+    - "Pasquale Davide Schiavone <pschiavo@iis.ee.ethz.ch>"
+    - "Germain Haugou <haugoug@iis.ee.ethz.ch>"
+    - "Francesco Conti <f.conti@unibo.it>"
+    - "Pirmin Vogel <vogelpi@lowrisc.org>"
+    - "Stefan Mach <smach@iis.ee.ethz.ch>"
+    - "Antonio Pullini <pullinia@iis.ee.ethz.ch>"
+    - "Gianmarco Ottavi <gianmarco.ottavi2@unibo.it>"
+    - "Manuel Eggimann <meggiman@iis.ee.ethz.ch>"
+    - "Luca Valente <luca.valente2@unibo.it>"
+
+dependencies:
+  L2_tcdm_hybrid_interco: { git: "git@github.com:pulp-platform/L2_tcdm_hybrid_interco.git", version: 1.0.0 }
+  adv_dbg_if:             { git: "git@github.com:pulp-platform/adv_dbg_if.git", version: 0.0.2 }
+  apb2per:                { git: "git@github.com:pulp-platform/apb2per.git", version: 0.1.0 }
+  apb_adv_timer:          { git: "git@github.com:pulp-platform/apb_adv_timer.git", version: 1.0.3 }
+  apb_fll_if:             { git: "git@github.com:pulp-platform/apb_fll_if.git", version: 0.1.3 }
+  apb_gpio:               { git: "git@github.com:pulp-platform/apb_gpio.git", rev: "0e9f142f2f11278445c953ad011fce1c7ed85b66" }
+  apb_node:               { git: "git@github.com:pulp-platform/apb_node.git", version: 0.1.1 }
+  apb_interrupt_cntrl:    { git: "git@github.com:pulp-platform/apb_interrupt_cntrl.git", version: 0.1.1 }
+  axi:                    { git: "git@github.com:pulp-platform/axi.git", version: 0.24.1 }
+  # axi_node:               { git: "git@github.com:pulp-platform/axi_node.git", version: 1.1.4 } # deprecated, replaced by axi_xbar (in axi repo)
+  axi_slice:              { git: "git@github.com:pulp-platform/axi_slice.git", version: 1.1.4 } # deprecated, replaced by axi_cut (in axi repo)
+  axi_slice_dc:           { git: "git@github.com:pulp-platform/axi_slice_dc.git", version: 1.1.3 } # deprecated, replaced by axi_cdc (in axi repo)
+  timer_unit:             { git: "git@github.com:pulp-platform/timer_unit.git", version: 1.0.2 }
+  common_cells:           { git: "git@github.com:pulp-platform/common_cells.git", version: 1.20.0 }
+  cluster_interconnect:   { git: "git@github.com:pulp-platform/cluster_interconnect.git", version: 1.1.1 }
+  fpnew:                  { git: "git@github.com:pulp-platform/fpnew.git", version: 0.6.5 }
+  jtag_pulp:              { git: "git@github.com:pulp-platform/jtag_pulp.git", rev: "v0.1" }
+  riscv:                  { git: "git@github.com:micprog/cv32e40p.git", rev: "pulpissimo-v3.4.1"} # To be updated to openhwgroup repository
```

Comment:
Any update on benderization of  cv32e40p? I would prefer to avoid letting the master branch of pulp_soc to point to a user repo.

Reply:
I guess ri5cy needs to be patched. We now have a fork in `pulp-platform` to maintain such patches. I suggest  the patch to go there.

Reply:
The changes to the Bender.yml file in ri5cy/cv32e40p are very limited, however updating this in the cv32e40p master would not improve the situation, as updating this in pulp may have a larger effect (core renaming, ...). Ibex uses a branch in the lowrisc repo to maintain pulp modifications, a fork in pulp-platform may be an option, I believe there is an issue open for this: https://github.com/pulp-platform/pulp_soc/issues/44

Reply:
I mean like this https://github.com/pulp-platform/cv32e40p/tree/pulpissimo-v3.4.0-rev1

Reply:
Looks good, I will make a pull request to that repo with my changes to the ri5cy/cv32e40p Bender.yml file and update the dependency here once merged and tagged.